pwem/objects/data.py

```diff
@@ -136,14 +136,18 @@ def __init__(self, **kwargs):
 
     def __str__(self):
         phaseShift = self.getPhaseShift() if self.hasPhaseShift() else 0
```

Comment:
I think now you can treat phaseShift the same as the other ones.

Reply:
Yep, done.